src/main/target/link/stm32_flash_split.ld

```diff
@@ -34,6 +34,15 @@ SECTIONS
     . = ALIGN(4);
   } >FLASH
 
+  /* System memory (read-only bootloader) interrupt vector */
+  .system_isr_vector (NOLOAD) :
+  {
+    . = ALIGN(4);
+    PROVIDE (system_isr_vector_table_base = .);
+    KEEP(*(.system_isr_vector)) /* Bootloader code */
```

Comment:
IMO no longer necessary, empty section with `system_isr_vector_table_base` should do

Reply:
Are these `ALIGN ` directives necessary or I could just set align for the whole section? There seems to be more than one way to achieve the same effect.